pages/index.html

```diff
@@ -4,9 +4,24 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
   <title>Document</title>
 </head>
 <body>
-  
+  <header>
+    <h1>Shaina's Website</h1>
+    <nav>
+      <a href="#">Home</a>
```

Comment:
Using `#` as the href ends up making a link that still looks clickable, but which doesn't go anywhere. It can often be helpful for web crawlers (as well as helping users understand where they are) to avoid putting a link around the text of the current page at all. Additional style may be required to make it look similar, but without the underline that an active link picks up.